packages/lib-classifier/src/plugins/drawingTools/experimental/models/tools/FreehandLineTool/FreehandLineTool.js

```diff
@@ -8,16 +8,19 @@ const FreehandLineTool = types
     type: types.literal('freehandLine')
   })
   .actions((self) => ({
-    createMark(mark) {
+    createMark(mark, points) {
```

Comment:
```suggestion
    createMark(mark, points = []) {
```

Reply:
Got rid of this in the mark creation and added the default to `initialize()` instead. Good catch.